src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SelectionRows/Header/Header.tsx

```diff
@@ -0,0 +1,98 @@
+import { useTranslation } from 'react-i18next';
+import { useFormContext } from 'react-hook-form';
+
+import { Svg } from 'shared/components';
+import { SelectEvent } from 'shared/types/event';
+import {
+  StyledClearedButton,
+  StyledFlexTopCenter,
+  StyledLabelBoldLarge,
+  variables,
+  theme,
+} from 'shared/styles';
+import { createArray } from 'shared/utils';
+import {
+  SelectionRowsItem,
+  DEFAULT_EMPTY_SELECTION_ROWS_OPTION,
+  DEFAULT_SELECTION_ROWS_SCORE,
+} from 'modules/Builder/features/ActivityItems/ItemConfiguration';
```

Comment:
i suppose in this case we should import it using relative paths to a specific file to avoid circular dependencies

Reply:
fixed